src/components/admin/form-components/withValidate.jsx

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 
 import { getDisplayName } from 'lib/higher-order-helpers';
 
+/**
+* This component wraps fields and handles the validation of the value inside of them
+* @param {func} validators - array of validation functions of the form which take the
```

Comment:
Missing a word at the end here

Reply:
Also when I said `of the form` I actually meant `of the shape` haha, like what their input and output is, but anything works